manage.py

```diff
@@ -59,6 +59,7 @@ def init_hq_python_path():
         'ptop_preindex',
         'sync_prepare_couchdb_multi',
         'sync_couch_views',
+        'celery',
```

Comment:
this essentially monkey patches all of hq for those management commands listed. we need to monkey patch for celery else gevent won't actually work